### PR TITLE
Final readme fixes

### PR DESCRIPTION
**Pull request by foxish**
@vishh @aronchick @jlewi

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,29 +88,15 @@ Once you have an external IP, you can proceed to visit that in your browser. The
...
+You can create new Tensorflow Training deployments by submitting a specification to the aforementioned API. For examples, look under the [tf-controller-examples/](https://github.com/google/kubeflow/tree/master/tf-controller-examples) directory.
 
-### Serve Model
-
-TODO(owensk)
-
-## Components
-
-### JupyterHub
+Refer to the README in the [tensorflow/k8s](https://github.com/tensorflow/k8s) repository for more information on using the TfJob controller to run TensorFlow jobs on K8s.
```
Comment (aronchick) [inline README.md]: I'm not fond of sending people to another repo to read up on how to use. Can we keep the instructions inline?

Comment (foxish) [reply README.md]: The instructions are a bit complex - @jlewi PTAL

Comment (foxish) [reply README.md]: Added some sample config but retained pointer to more detailed docs. @aronchick PTAL.

**Comment by foxish** [comment]
Merging. @vishh, if you have any concerns, please let me know. Will address in follow-up.

**Pull request merged**